modules/ethereum/src/benchmarking.rs

```diff
@@ -55,6 +60,163 @@ benchmarks! {
 
 	}: import_unsigned_header(RawOrigin::None, header, None)
 	verify {
-		assert_eq!(BridgeStorage::<T>::new().best_block().0.number, 1);
+		let storage = BridgeStorage::<T>::new();
+		assert_eq!(storage.best_block().0.number, 1);
+		assert_eq!(storage.finalized_block().number, 0);
+	}
+
+	// Our goal with this bench is to try and see the effect that finalizing difference ranges of
+	// blocks has on our import time. As such we need to make sure that we keep the number of
+	// validators fixed while changing the number blocks finalized (the complixity parameter) by
+	// importing the last header.
+	import_unsigned_finality {
```

Comment:
One last thing I forgot to add to my top-level comment - imo `weight(import_signed_headers)` is actually `weight(import_unsigned_header) * len(headers) + some_constant`, where `some_constant` stands for recovering signature + iterating headers + invoking callbacks (which are noop in our runtime - not sure how to measure that in general case?). So we may only introduce benchmarks for unsigned version and use results for signed version.